README

```diff
@@ -0,0 +1,12 @@
+# WeatherApp
```

Comment:
this Readme should be .md. You can use the default README.md instead of deleting it and use the correct format for that

Reply:
Done thx